docs/source/testing.rst

```diff
@@ -92,46 +102,80 @@ settings.local.yaml:
           password: password
    
 
-Smoke test:
-::
+Smoke test::
+
     $ cd koku/testing
     $ ./run_smoke_tests.sh
     
+
+tox
+===
+
+To run a specific tox test, you may run a command below, replacing everything after the :code:`--` with your specific modules and files::
+
+    tox -e py36 -- masu.test.external.downloader.azure.test_azure_services.AzureServiceTest
+
+This will selectively run only the AzureServiceTest module
+The argument can be vague or specified further, for example the following arguments would separately be considered valid:
+
+    - masu.test.external
+    - masu.test.external.downloader.azure.test_azure_services.AzureServiceTest.specific_test
+
+Debugging Options
+=================
+
+Running PDB in dockerized koku
```

Comment:
Let's avoid "dockerized" and just use "in a container" or "in Docker" instead. I prefer "in a container" just because it's a bit more agnostic. Docker and Podman both run our containers, and OpenShift is likely to convert over to Podman completely at some point.